library/cirros

```diff
@@ -1,4 +1,5 @@
-# maintainer: Eric Windisch <ewindisch@docker.com> (@ewindisch)
+# maintainer: Eric Windisch <eric@windisch.us> (@ewindisch)
 
-latest: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
+latest: git://github.com/ewindisch/docker-cirros@fd9aa114c465237518535d3545d4f9c4bc1d0aa5
+0.3.4: git://github.com/ewindisch/docker-cirros@fd9aa114c465237518535d3545d4f9c4bc1d0aa5
 0.3.3: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
```

Comment:
Did you want to drop `0.3.3`?  The image will stay on the Docker Hub; this list controls rebuilds and the list of "Supported Versions" on the Hub description.